frontend/src/store/auth/reducer.ts

```diff
@@ -1,13 +1,15 @@
 import { createReducer } from '@reduxjs/toolkit';
 import { DataStatus } from 'common/enums/enums';
-import { signUp } from './actions';
+import { signUp, User } from './actions';
```

Comment:
```suggestion
import { signUp } from './actions';
```
why? `common/types/types` ?